src/app/(private)/admin-registration-applications/[id]/page.tsx

```diff
@@ -1,8 +1,222 @@
 'use client'
-export default function Page({
-  params,
-}: {
-  params: { slug: string; id: string }
-}) {
-  return <div>{params.id}</div>
+
+import { useEffect, useState } from 'react'
+import { getAdminRegistrationApplication } from '@/modules/adminRegistrationApplications/service'
+import { Input } from '@/components/ui/input'
+import { AdminRegistrationApplication } from '../../../../modules/adminRegistrationApplications/constants/types'
+import { formatDate } from '@/app/hocs/formatDate'
+import Status from '@/modules/adminRegistrationApplications/components/status'
+import { FormItem, FormField, FormControl, FormLabel, Form } from '@/components/ui/form'
+import { useForm } from 'react-hook-form'
+import { zodResolver } from '@hookform/resolvers/zod'
+import { FormSchema } from '@/modules/adminRegistrationApplications/constants/formSchema'
+
+const Page = ({ params }: { params: { slug: string; id: string } }) => {
+  const form = useForm({
+    resolver: zodResolver(FormSchema),
+  })
+  const { control } = form
+
+  const [adminRegistrationApplicationDetails, setAdminRegistrationApplicationDetails] = useState<AdminRegistrationApplication | null>(null)
+  const [loading, setLoading] = useState(true)
+  const [error, setError] = useState<any>(null)
+
+  useEffect(() => {
+    const fetchDetails = () => {
+      getAdminRegistrationApplication(params.id)
+        .then((response) => {
+          if (response.data.isSuccess) {
+            setAdminRegistrationApplicationDetails(response.data.response)
+          } else {
+            setError(new Error('Failed to fetch data'))
+          }
+          setLoading(false)
+        })
+        .catch((err) => {
+          setError(err)
+          setLoading(false)
+        })
+    }
+    fetchDetails()
+  }, [params.id])
+
+  return (
+    <div className="form-container p-6 bg-white dark:bg-gray-800 rounded-md shadow-md text-black dark:text-white">
+      {loading && <div>Loading...</div>}
+      {error && <div>Error: {error.message}</div>}
```

Comment:
Error v.b gösterimlerde , daha kararlı ve kullanıcı dostu yapılar için div kullanmak yerine [shadcn/Alert](https://ui.shadcn.com/docs/components/alert) componentı tercih etmemiz daha sağlıklı olur.